python/paddle/fluid/dygraph/varbase_patch_methods.py

```diff
@@ -226,6 +226,27 @@ def grad(self):
 
         return self.gradient()
 
+    @property
+    def version(self):
```

Comment:
version is a common name so it is not good here. For example, protobuf has versions. Can we change it to "inplace_version" for inplace operation or "modify_version" if you make it extendable for future modify

Reply:
Also change some corresponding names